### ISSUE TITLE
BUG: When sorting by single lift it should go off of that lifts wilks score in the event of a tie

### ISSUE THREAD

**Issue by TrystanOakley** [sstangl/openpowerlifting#935]
When sorting by single lifts and lifters are tied, like if two lifters have the same bench press, it is secondary sorting off of their total wilks score instead of that single lifts wilks score. Essentially when sorting by single lifts the ranking should be determined by bodyweight in the event of a tie I suppose. Or single lift wilks if you want to get that deep into it because I suppose if a man and woman both benched the same amount and weighed the same she should still be ranked higher based off wilks so. @MichaelVessia

**Comment by TrystanOakley** [comment]
Also, in the event that two or more lifters are tied and wilks is looked at, if wilks is also a tie then Wilks should secondary sort by date. If the date is the same then... I have no idea

**Comment by MichaelVessia** [comment]
Yes, if the single lift is tied it will currently break the tie by overall wilks.  Without looking into it, I'm not sure how much work it would be to sort by a single lift wilks.  If it's not trivial it might make more sense to implement sorting exactly how we'd like with the new `server/` implementation, rather than invest effort into changing it on the current site.

**Comment by MichaelVessia** [comment]
Ah, that makes sense.  You can assign to me

**Comment by TrystanOakley** [comment]
I believe this can be closed correct?

**Comment by sstangl** [comment]
Oh, it's not quite a one-liner, since the bodyweightKg should secondary-sort in the opposite order.
